src/docs/robonomics-hass-integration.md

```diff
@@ -22,20 +22,18 @@ Press "Take Control" one more time:
 
 Now you can install the Robonomics integration. To do so, follow these steps:
  
-{% roboWikiVideo {videos:[{src: 'QmQp66J943zbF6iFdkKQpBikSbm9jV9La25bivKd7cz6fD', type: 'mp4'}], attrs:['loop', 'controls', 'autoplay']} %}{% endroboWikiVideo %}
 
 1. In the web interface of Home Assistant go to `Settings` -> `Device & Services` and press `ADD INTEGRATION`. Search for `Robonomics`.
 
-2. Click on Robonomics and fill in the configuration:
+2. Click on Robonomics, upload your setup file, and enter password for the `CONTROLLER` account. Instructions on how to create the setup file can be found [here](/docs/sub-activate/?topic=smart-home#setup-your-subscription)
```

Comment:
may be to write - "upload your setup file `robonomics.app-settings-<subscirption-name>-server.json`, and enter ..."